adbe/adb_helper.py

```diff
@@ -207,3 +207,15 @@ def _check_for_device_not_found_error(stderr_data):
 
 def toggle_screen():
     return execute_adb_shell_command2("input keyevent KEYCODE_POWER")
+
+
+def set_device_id(device_id):
```

Comment:
The PR title and the code change doesn't seem to match

Reply:
Indeed, this pull request can be ignored as it was a mistake using the wrong branch on local repository.